packages/p2p/src/components/trade-badge/trade-badge.scss

```diff
@@ -0,0 +1,27 @@
+.trade-badge {
+    align-items: center;
+    border-radius: 3.4rem;
+    cursor: default;
+    display: flex;
+    height: 1.8rem;
+    justify-content: center;
+    margin-right: 0.4rem;
```

Comment:
Suggestion: You may need to add margin top and bottom to make sure the badges don't stack up on each other!